oauth2client/service_account.py

```diff
@@ -240,6 +240,8 @@ def _from_p12_keyfile_contents(cls, service_account_email,
         """
         if private_key_password is None:
             private_key_password = _PASSWORD_DEFAULT
+        if crypt.Signer is not crypt.OpenSSLSigner:
```

Comment:
Expand the diff, you'll see this is only in the `_from_p12_keyfile_contents` helper